### ISSUE TITLE
Releases are currently broken

### ISSUE THREAD

**Issue by garethjevans** [jenkinsci/conventional-commits-plugin#67]
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-deploy-plugin:2.8.2:deploy (default-deploy) on project conventional-commits: Failed to deploy artifacts: Could not transfer artifact io.jenkins.plugins:conventional-commits:hpi:0.3.0 from/to maven.jenkins-ci.org (https://repo.jenkins-ci.org/releases/): authorization failed for https://repo.jenkins-ci.org/releases/io/jenkins/plugins/conventional-commits/0.3.0/conventional-commits-0.3.0.hpi, status: 403 Forbidden -> [Help 1]
[ERROR] 
```

This looks like its an issue with repository-permissions-updater pushing out credentials that are out of sync with artifactory.

**Comment by garethjevans** [comment]
@olblak do you know anything about this?

**Comment by olblak** [comment]
We are investigating issues with Jfrog on repo.jenkins-ci.org, I don't know it if its related

**Comment by olblak** [comment]
@garethjevans  Ha I missed that this issue was created two days ago, indeed we identify the root cause and yesterday we were able to do a release as you can see [here](https://github.com/jenkinsci/conventional-commits-plugin/runs/3074797064?check_suite_focus=true)

For some reason, a previous Github action run published the release 0.3.0 on repo.jenkins-ci.org but failed to publish the release and so creating the correct tag 0.3.0 on this repository. Every cd job triggered after the release 0.3.0 failed because they tried t override artifacts published on repo.jenkins-ci.org
